setup.py

```diff
@@ -167,7 +167,7 @@ def package_files(prefix: str, directory: str, extension: str):
 composer_data_files += package_files("composer", "yamls", ".yaml")
 composer_data_files += package_files("composer", "algorithms", ".json")
 
-setup(name="mosaicml",
+setup(name=os.environ.get('COMPOSER_PACKAGE_NAME', "mosaicml"),
```

Comment:
would you want to print/log the package name so that automated builds/logs are easier to debug?